tests/Avalonia.Skia.UnitTests/Media/FontManagerTests.cs

```diff
@@ -87,5 +86,33 @@ public void Should_Return_False_For_Unregistered_FontCollection_Uri()
                 Assert.False(result);
             }
         }
+
+        [Fact]
+        public void Should_Load_Embedded_Font_With_Wrong_CharacterCasing()
+        {
+            using (UnitTestApplication.Start(TestServices.MockPlatformRenderInterface.With(fontManagerImpl: new FontManagerImpl())))
+            {
+                var result = FontManager.Current.TryGetGlyphTypeface(new Typeface("resm:Avalonia.Skia.UnitTests.Assets?assembly=Avalonia.Skia.UnitTests#noto mOnO"), out var glyphTypeface);
+
+                Assert.True(result);
+
+                Assert.Equal("Noto Mono", glyphTypeface.FamilyName);
+            }
+        }
+
+        [Fact]
+        public void Should_Load_Embedded_DefaultFontFamily()
+        {
+            using (UnitTestApplication.Start(TestServices.MockPlatformRenderInterface.With(fontManagerImpl: new FontManagerImpl())))
+            {
+                AvaloniaLocator.CurrentMutable.BindToSelf(new FontManagerOptions { DefaultFamilyName = s_fontUri });
```

Comment:
Locator registrations should be inside of locator scope for unit tests: https://github.com/AvaloniaUI/Avalonia/blob/7430ffe6c2bfeb877d71f9af49cba18284022d25/tests/Avalonia.Markup.Xaml.UnitTests/Xaml/StyleIncludeTests.cs#L284

So it can be reset back after exiting the method